Seminar-IS/Seminar02/Pract 02/task03.cpp

```diff
@@ -0,0 +1,168 @@
+#include <iostream>
+#include <fstream>
+#include <string.h>
+
+using namespace std;
+
+#pragma warning(disable : 4996)
+
+enum class Tyres {
+    Soft=45,
+    Medium=50,
+    Hard=60,
+    Intern=40,
+    Wet=35
+};
+
+const size_t MAX_NAME_LENGTH = 65;
+const size_t MAX_TYRE_LENGTH = 10;
+const size_t MAX_ROW_SIZE = 70;
+const char FILE_NAME[] = "result.csv";
+
+struct Bolid {
+    double tyreWearCoefficient;
+    size_t maxLoops;
+    Tyres tyres;
+    short int yearOfManafacture;
+    char name[MAX_NAME_LENGTH];
+    bool inRepairStation;
+};
+
+void readTyres(Tyres& tyres) {
+    char type[MAX_TYRE_LENGTH];
+    cin.getline(type, MAX_TYRE_LENGTH);
+    if (strcmp(type, "Soft") == 0) {
+        tyres = Tyres::Soft;
+    }
+    if (strcmp(type, "Medium") == 0) {
+        tyres = Tyres::Medium;
+    }
+    if (strcmp(type, "Hard") == 0) {
+        tyres = Tyres::Hard;
+    }
+    if (strcmp(type, "Intern") == 0) {
+        tyres = Tyres::Intern;
+    }
+    if (strcmp(type, "Wet") == 0) {
+        tyres = Tyres::Wet;
+    }
+    delete[] type;
```

Comment:
Missed a delete here. And why you removed the else-ifs?

Reply:
> And why you removed the else-ifs?

I've made some changes using the website but forgot to make them on the local files. Then I made other changes in the local files and uploaded them manually which reverted the web changes.